lib/pdk/cli/util/option_validator.rb

```diff
@@ -23,6 +23,7 @@ def self.valid_module_name?(string)
           !(string =~ %r{\A[a-z][a-z0-9_]*\Z}).nil?
         end
         singleton_class.send(:alias_method, :valid_task_name?, :valid_module_name?)
+        singleton_class.send(:alias_method, :valid_plan_name?, :valid_module_name?)
```

Comment:
Plans also require checking the name against the list of reserved words, will need to account for that somehow.